index.html

```diff
@@ -0,0 +1,299 @@
+                    </li>
+                    <li class="news-item life__news-item">
+                        <div class="news-item__date">
+                            <div class="news-item__day">06</div>
+                            <div class="news-item__month">Jan</div>
+                        </div>
+                        <div class="news-item__article">
+                            <h4 class="h4-like news-item__title"><a class="link" href="#">Tellus Mattis Pelle</a></h4>
+                            <span class="news-item__comments">07 Comments</span>
+                            <p lang="en" class="prh news-item__about">
+                                <a class="link" href="#">Morbi in sem quis dui placerat ornare. Pellentesque odio nisi, euis&shymod in, 
+                                                            pharetra a, ultricies in, diam. Sed arcu
+                                    <span class="news-item__more-icon icon-arrow-circle-right"></span>
+                                </a>
+                            </p>
+                        </div>
+                    </li>
+                    <li class="news-item life__news-item ">
+                        <div class="news-item__date">
+                            <div class="news-item__day">05</div>
+                            <div class="news-item__month">Jan</div>
+                        </div>
+                        <div class="news-item__article">
+                            <h4 class="h4-like news-item__title"><a class="link" href="#">Phasellus Lacina</a></h4>
+                            <span class="news-item__comments">22 Comments</span>
+                            <p lang="en" class="prh news-item__about">
+                                <a class="link" href="#">Lospendisse ac urna. Etiam pellentesque mauris ut lectus. Nunc
+                                                                tellus ante, mattis eget, gravida vitae, ultricies atinvio
+                                    <span class="news-item__more-icon icon-arrow-circle-right"></span>
+                                </a>
+                            </p>
+                        </div>
+                    </li>
+                </ul>
+                <ul class="life__says-list">
+                    <li class="says-item life__says-item">
+                        <div class="says-item__comment">
+                            <p lang="en" class="prh says-item__what">
+                                “ Sed egestas, ante et vulputate volutpat, eros pede 
+                                semper est, vitae luctus metus libero eu augue. Morbi purus libero, faucibus adipiscing’
+                            </p>
+                            <p class="says-item__who">- John Doe</p>
+                        </div>
+                        <div class="circle-icon says-item__icon">
+                            <img class="says-item__icon-img" src="./resources/images/comments.svg">
```

Comment:
Always be consistent! Use some one format for src paths. Earlier you used just "resources/...".